contrib/olivere/elastic/elastictrace_test.go

```diff
@@ -271,6 +271,57 @@ func TestQuantize(t *testing.T) {
 	}
 }
 
+func TestResourceNamerSettings(t *testing.T) {
+	staticName := "static resource name"
+	staticNamer := func(url, method string) string {
+		return staticName
+	}
+
+	t.Run("default", func(t *testing.T) {
+		mt := mocktracer.Start()
+		defer mt.Stop()
+
+		tc := NewHTTPClient()
+		client, err := elasticv5.NewClient(
+			elasticv5.SetURL("http://127.0.0.1:9200"),
+			elasticv5.SetHttpClient(tc),
+			elasticv5.SetSniff(false),
+			elasticv5.SetHealthcheck(false),
+		)
+		assert.NoError(t, err)
+
+		_, err = client.Get().
+			Index("logs_2016_05/event/_search").
+			Type("tweet").
+			Id("1").Do(context.TODO())
+
+		span := mt.FinishedSpans()[0]
+		assert.Equal(t, "GET /logs_?_?/event/_search/tweet/?", span.Tag(ext.ResourceName))
+	})
+
+	t.Run("custom namer", func(t *testing.T) {
```

Comment:
Can you please name this test `custom`? I think spaces between words or sentences look pretty ugly when running tests with `-v` because the `go` command adds underscores to replace them, which to me is an indication that test aren't supposed to be named using sentences like in other languages.